docs/source/usage/iss/iss_vignette.rst

```diff
@@ -0,0 +1,163 @@
+In-Situ Sequencing
+==================
+
+This in-situ sequencing (ISS) experiment has 16 fields of view that measure 4 channels on 4 separate
+imaging rounds.
+
+Downloading Data
+----------------
+
+Like all starfish data, this experiment is hosted on amazon web services. Once formatted,
+experiments can be downloaded on-demand into starfish. However, downloading the raw data requires
+that you access the data from an aws account.
+
+For the purposes of this vignette, we will format only two of the 16 fields of view. To download the
+data, you can run the following commands:
+
+.. code-block:: bash
+
+    mkdir -p iss/raw
+    aws s3 cp s3://czi.starfish.data.public/browse/raw/20180820/iss_breast/ iss/raw/ \
```

Comment:
This leads to `fatal error: Unable to locate credentials`. Workaround for the general public is to add `--no-sign-request`. Tested via:

```
FROM alpine:3.6
RUN apk -v --update add \
        python \
        py-pip \
        groff \
        less \
        mailcap \
        && \
    pip install --upgrade awscli==1.14.5 s3cmd==2.0.1 python-magic && \
    apk -v --purge del py-pip && \
    rm /var/cache/apk/*
WORKDIR /data
ENTRYPOINT ["aws"]
```

Reply:
Awesome, is that a general work-around for use of links with public permissions?

Reply:
`aws s3 --no-sign-request ...` has worked for every read operation I've tried so far (e.g. `cp`, `ls`, `sync`)